### DISCUSSION TITLE
Strugling with favicon

### DISCUSSION THREAD

**Discussion by rogerio-romao** [quasarframework/quasar#12387] [General - Components / Directives / etc]
Hi,

I have used the Icongenie CLI and it successfully created the icons and copied them to statics/icons.
I then copied the tags it gave me and put them on the html head.
I can see the files exist and are valid images.
However, i keep getting 404s on the browser.

The documentation says these types of assets should be in the /public folder, however there is only /statics and that is where Icongenie puts them. Any tips on what could be happening?

**Comment by yusufkandemir** [comment]
If you are using Quasar v1 with `@quasar/app` v1, it will use `statics` folder. But, I am assuming you are using a more up-to-date version because it's not working. Please let us know which versions of packages you are using when asking questions to make it easier for both sides.

1. Make sure you are not using the legacy app extension version.
2. Make sure you are using an up-to-date version of Icon Genie CLI, running `npm install -g @quasar/icongenie` should update it.
3. Make sure you are executing the generate command while on the project root folder. (e.g. where the `quasar.conf.js` is)
4. Make sure you have a `public` folder at the root of your project.

**Comment by rogerio-romao** [comment]
Thank you for the help.

I am using Quasar 2.0 and Icongenie 2.4.0, and did execute the command on the project root folder.
I think I may have at some point in the past manually deleted the public folder, as I was having some issues with the dev server.
I have now created the public folder and put the icons there, and everything works in development.

However, in production, I am still not getting the files. It's deployed on Vercel and I followed the documentation regarding deploying there, including creating the vercel.json
When I inspect the network requests, the icon files are being requested as text/html type, which I suspect is the problem, even though they are set as image/ico and image/png on the html.

I will try to figure that out, unless you have any idea why that might be the case.

**Comment by rogerio-romao** [comment]
I think it may have something to do with the quasar meta plugin, which I am using.
I've found reports on Vercel about issues with React's Helmet package, which is similar.
Maybe I can send the favicon links through meta plugin instead of directly on the html?

**Comment by joakweb** [comment]
I'm experiencing a similar issue, generated with icongenie, everything works fine during quasar dev, but when I build and try to run with quasar serve, I'm getting 404s all around, like: 

GET
http://127.0.0.1:4020/favicon.ico
[HTTP/1.1 404 Not Found 2ms]

--My setup:

node18.20.4
"quasar": "^2.17.7",
"@quasar/extras": "^1.16.17",
"@quasar/app-vite": "^1.9.0",
Linux Mint 21.3

Even scaffolding a new project produced the same issue. I have a feeling it's something trivial but I'm at a loss here.

edit. Solved. Indeed it was trivial. I just needed to set the publicPath to the same distDir, was only setting the former before.
